### ISSUE TITLE
it is not possible to browse the contact suggestion list in the add someone to microsoft teams chat 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12561]
### Steps to reproduce:
in nvda's explore mode options is set to:
Auto focus mode when focus changes
marked

open the teams
in the chat tab start a new chat with a person
now walk with tabe and press the button
add people
in the search field enter a name, email or brand start typing a person's name as an example
When nvda announces that search results have been updated
navigate with the down arrows as directed by the teams help read by nvda to navigate the results 
### Actual behavior:
nvda goes into browse mode and search results are not read.
By pressing nvda + space to go to focus mode, and navigating with the down arrows the nvda section announces and does not read the results.
This seems to happen only in this window (add people) of the teams, in the suggestions displayed in the new chat window the suggestions are read normally. 
there is another problem with this window to add people to the chat, even if there are results available nvda announces that there are no results available. 
### Expected behavior:
nvda should remain in focus mode and when walking with the arrows below you should read the suggested results.
Nvda should also not announce that there are no results even if there are results available. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-23112,0d98162d

#### Windows version:
10.0.19043.1055
#### Name and version of other software in use when reproducing the issue:
Teams.exe, Microsoft Teams 1.4.00.11161
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[teams add.txt](https://github.com/nvaccess/nvda/files/6673009/teams.add.txt)

**Comment by seanbudd** [comment]
Is this still an issue?

**Comment by fernando-jose-silva** [comment]
at this point with nvda alpha and most updated teams everything works fine.
I believe we can close for the moment
